tests/export/html/test_heading.py

```diff
@@ -506,10 +523,9 @@ def test_heading_as_list_following_bare_paragraph_plus_list(self):
 
         expected_html = '''
             <ol class="pydocx-list-style-type-decimal">
-                <li>foo</li>
+                <li>foo<br />bare paragraph</li>
```

Comment:
Same thing here. I assume we are rolling them in because they have the same `numId`